tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 minversion = 3.7.0
 envlist =
-    py{27,36,37}-ansible{24,25,26,27}-{functional,unit}
-    py{27,36,37}-lint
+    py{26,27,35,36,37}-ansible{24,25,26,27}-{functional,unit}
```

Comment:
https://docs.ansible.com/ansible/devel/dev_guide/developing_python_3.html#minimum-version-of-python-3-x-and-python-2-x

> On the controller we support Python 3.5 or greater and Python 2.7 or greater.
> Module-side, we support Python 3.5 or greater and Python 2.6 or greater.

Reply:
done my homework